lametro/models.py

```diff
@@ -485,21 +484,6 @@ def board_event_minutes(self):
                 return doc.url
 
 
-class LAMetroEventMedia(EventMedia):
```

Comment:
this looks like a work around a data import issue, and I think it will not be necessary. I'll keep track of it though.

Reply:
didn't need this model.